charts/foundations/templates/klum.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if or (not .Values.foundations) (not .Values.foundations.disable) (not .Values.foundations.disable.klum) }}
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: GitRepository
+metadata:
+  name: klum
+spec:
+  interval: 30m
+  url: https://github.com/jadolg/klum
+  ref:
+    tag: v0.8.1
+  ignore: |
+    # exclude all
+    /*
+    # include deploy.yaml
+    !/deploy.yaml
+---
+# Makes sure nothing crashes while keeping all defaults empty and the code simple
+# {{- if not .Values.klum }}{{- set .Values "klum" (dict) }}{{ end }}
```

Comment:
leftovers

Reply:
No; this makes the rest of the code much simpler by making sure the `klum` object is not nil but rather empty;
The default configuration is bad so I'm replacing all env variables, and it would make the code much more complex to check if klum exists and the variable is set every time.

Reply:
oh ok, i misinterpreted it as commented out